capstone/src/main/java/com/google/univiz/collegescorecarddata/SearchData.java

```diff
@@ -0,0 +1,8 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface CollegeScoreCard {
+  /** Search Feature Functions */
+  List<UniversityData> getCollegeSuggestions(String currName);
```

Comment:
Some thoughts here:
* what does the `curr` in `currName` mean?
* Based on the scope of `UniversityData` is shaping up - this is going to pull back a lot of data. Per this [API guidance](https://github.com/RTICWDT/open-data-maker/blob/master/API.md#limiting-returned-fields-with-fields) - we should be heavily restricting fields that come back from the server. I think there should be a light weight class for autocomplete that doesn't have all of the data that UniversityData has.

Reply:
ah I see-I can just return a list of college names